app/src/main/java/io/github/droidkaigi/confsched2018/data/db/ContributorDatabase.kt

```diff
@@ -0,0 +1,10 @@
+package io.github.droidkaigi.confsched2018.data.db
+
+import io.github.droidkaigi.confsched2018.data.api.response.Contributor
+import io.github.droidkaigi.confsched2018.data.db.entity.ContributorEntity
+import io.reactivex.Flowable
+
+interface ContributorDatabase {
+    fun getAllContributors(): Flowable<List<ContributorEntity>>
```

Comment:
I think `getAll()` is enough because interface name includes `Contributor`.
What do you think?

Reply:
Right. 
I'll fix it!